### PR TITLE
add more yamls for active sampler learning experiments

### PR DESCRIPTION
**Pull request by tomsilver**
bumpy_cover_num_online_transitions_perc_solved
shifted_bumpy_cover_num_online_transitions_perc_solved
regional_bumpy_cover_num_online_transitions_perc_solved

### CODE REVIEW

File: scripts/configs/active_sampler_learning.yaml [hunk 1]
```diff
@@ -1,4 +1,80 @@
...
+  bilevel_plan_without_sim: "True"
+  max_initial_demos: 0
+  num_train_tasks: 1000
+  num_test_tasks: 100
+  max_num_steps_interaction_request: 50
+  sampler_mlp_classifier_max_itr: 100000
+  mlp_classifier_balance_data: False
+  pytorch_train_print_every: 10000
+  active_sampler_learning_use_teacher: False  # NOTE
+  online_nsrt_learning_requests_per_cycle: 10  # NOTE
```
Comment (NishanthJKumar) [inline scripts/configs/active_sampler_learning.yaml:27]: What are the two `NOTE` comments here for?

File: scripts/configs/active_sampler_learning.yaml [hunk 2]
```diff
@@ -34,8 +110,8 @@ FLAGS:
   sampler_mlp_classifier_max_itr: 100000
   mlp_classifier_balance_data: False
   pytorch_train_print_every: 10000
-  active_sampler_learning_use_teacher: False # NOTE
-  online_nsrt_learning_requests_per_cycle: 1
+  active_sampler_learning_use_teacher: False  # NOTE
+  online_nsrt_learning_requests_per_cycle: 1  # NOTE
```
Comment (NishanthJKumar) [inline scripts/configs/active_sampler_learning.yaml:114]: Again, why the `NOTE` comments?

**Review by NishanthJKumar — APPROVED** [review]
LGTM, with minor comments!

File: scripts/configs/active_sampler_learning.yaml (hunk 1 above)
Comment (tomsilver) [reply scripts/configs/active_sampler_learning.yaml:27]: these are just things that change meaningfully between experiments and might be easy to miss

**Pull request merged**